docs/build/guides/basics/send-and-receive-payments.mdx

```diff
@@ -7,90 +7,221 @@ description: Learn to send payments and watch for received payments on the Stell
+
+Additionally, there are two main options to interact with the Stellar network: the [Horizon API](../../../data/apis/horizon/README.mdx) and [Stellar RPC](../../../data/apis/rpc/README.mdx). Generally speaking, you should use RPC unless you need features that only Horizon provides. If your use case is limited to making payments, RPC should be sufficient. Regardless, each payment example will be broken down to show how to perform such payments with either option.
 
 <Alert>
 
 In the following code samples, proper error checking is omitted for brevity. However, you should _always_ validate your results, as there are many ways that requests can fail. You should refer to the guide on [Error Handling](../../../data/apis/horizon/api-reference/errors/error-handling.mdx) for tips on error management strategies.
 
 </Alert>
 
-## Send a Payment
+## Using the Payment Operation
+
+### Send a Payment
 
-Stellar stores and communicates transaction data in a binary format called [XDR](../../../learn/fundamentals/data-format/xdr.mdx), which is optimized for network performance but unreadable to the human eye. Luckily, [Horizon](../../../data/apis/horizon/README.mdx), the Stellar API, and the [Stellar SDKs](../../../tools/sdks/README.mdx) convert XDRs into friendlier formats. Here’s how you might send 10 lumens to an account:
+When sending payments on Stellar, you build a transaction with a payment operation, sign it, and submit it to the network. Stellar stores and communicates transaction data in a binary format called [XDR](../../../learn/fundamentals/data-format/xdr.mdx), which is optimized for network performance but unreadable to the human eye. Luckily, [Horizon API](../../../data/apis/horizon/README.mdx), [Stellar RPC](../../../data/apis/rpc/README.mdx), and the [Stellar SDKs](../../../tools/sdks/README.mdx) convert XDRs into friendlier formats. Here’s how you might send 10 lumens to an account:
+
+#### Using the Stellar RPC
 
 <CodeExample>
 
 ```js
-var StellarSdk = require("stellar-sdk");
-var server = new StellarSdk.Horizon.Server(
+import * as StellarSdk from "stellar-sdk";
+
+// Initialize Stellar RPC server for testnet
+const rpc = new StellarSdk.rpc.Server("https://soroban-testnet.stellar.org");
+
+// Initalize the source account's secret key and destination account ID.
+// The source account is the one that will send the payment, and the destination account
+// is the one that will receive the payment.
+const sourceKeys = StellarSdk.Keypair.fromSecret(
+  "SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4"
+);
+const destinationId =
+  "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5";
+
+// First, check to make sure that the destination account exists.
+await rpc.getAccount(destinationId).catch((error) => {
+  if (error instanceof StellarSdk.NotFoundError) {
+    throw new Error("The destination account does not exist!");
+  } else throw error;
+});
```

Comment:
Can we use `try/catch` here for the `NotFoundError`? Same for the other instances of `.catch`